php/libraries/Visit.class.inc

```diff
@@ -0,0 +1,64 @@
+<?php
+/**
+ * Describe a LORIS visit
+ *
+ * PHP Version 7
+ *
+ * @category Model
+ * @package  Visits
+ * @author   Xavier Lecours Boucher <xavier.lecoursboucher@mcgill.ca>
```

Comment:
I dont think so

Reply:
Give back to Cesar what belong to Cesar. Xavier wrote this part

Reply:
sorry @xlecours